models/intel/person-attributes-recognition-crossroad-0234/description/person-attributes-recognition-crossroad-0234.md

```diff
@@ -0,0 +1,56 @@
+# person-attributes-recognition-crossroad-0234
+
+## Use Case and High-Level Description
+
+This model presents a person attributes classification algorithm analysis scenario.
+The model consists of the ResNet-50 backbone and a head, which returns for an input image with a pedestrian 7 values
+that are probabilities of the corresponding 7 attributes.
+
+## Specification
+
+| Metric                | Value                                                                                                 |
+|-----------------------|-------------------------------------------------------------------------------------------------------|
+| Pedestrian pose       | Standing person                                                                                       |
+| Occlusion coverage    | <20%                                                                                                  |
+| Min object width      | 80 pixels                                                                                             |
+| Supported attributes  | `is_male`, `has_bag`, `has_hat`, `has_longsleeves`, `has_longpants`, `has_longhair`, `has_coat_jacket`|
+| GFlops                | ???                                                                                                   |
+| MParams               | ???                                                                                                   |
+| Source framework      | PyTorch\*                                                                                             |
+
+
+## Accuracy
+
+| Attribute         |  F1   |
+|-------------------|-------|
+| `is_male`         | 0.92  |
+| `has_bag`         | 0.44  |
+| `has_hat`         | 0.74  |
+| `has_longsleeves` | 0.45  |
+| `has_longpants`   | 0.89  |
+| `has_longhair`    | 0.84  |
+| `has_coat_jacket` |  NA   |
```

Comment:
Do I right understand that result  is not available due to lack of images for this class in validation set?

Reply:
Hmmm... no, `has_hat` is an attribute with even smaller number of images.    
Nevertheless, at the moment accuracy_checker does not check the attribute `has_coat_jacket` -- may be due to historical reason